### PR TITLE
Test1

### PR DESCRIPTION
**Pull request by mnjaggah**
TestCases for room creation @kevintumbo @gdp254 @jomomg

### CODE REVIEW

**Review by kevintumbo — APPROVED** [review]

**Comment by jomomg** [comment]
* I don't think you should test whether `self.office_space` is an instance of `OfficeSpace` class since you have already explicitly made it an instance in the `setUp` function. So the test will never fail. 
* Same for `self.living_space`
* Otherwise, I think the rest is OK

**Comment by jomomg** [comment]
```Python
def test_person_type(self):
	person = Person() # You don't need this since you've already instantiated in the 'setUp' function
	post = Person.position # Typo: should be 'post = self.person.position'
	self.assertEqual('FELLOW' or 'STAFF', post,  
			msg = 'Wrong position specified') 
```

**Review by kevintumbo — CHANGES_REQUESTED** [review]
`person = Person()   #remove since already initialized in set up'

File: test_cases/tests.py [hunk 1]
```diff
@@ -60,8 +60,7 @@ def setUp(self):
 		self.staff = Staff()
 
 	def test_person_type(self):
-		person = Person()
-		post = Person.position
+		post = self.person.position
```
Comment (mnjaggah) [inline test_cases/tests.py]: Fixed test definition as suggested by @kevintumbo @jomomg

File: test_cases/tests.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+			msg ='Maximum capacity is 6 occupants')
+
+		self.assertEqual(self.office_space.room_type, "OFFICE",
+			msg = "Room created should be of type OFFICE")
+
+
+	def test_officespace_accepts_str(self):
+		self.assertRaises(ValueError, self.office_space.create_new, 111)
+
+
```
Comment (GDP254) [inline test_cases/tests.py]: 3 spaces to the next method. This is inconsistent with your previous spacing.

File: test_cases/tests.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+	def test_officespace_size_type(self):
+		self.assertEqual(self.office_space.size , 6 ,
+			msg ='Maximum capacity is 6 occupants')
+
+		self.assertEqual(self.office_space.room_type, "OFFICE",
+			msg = "Room created should be of type OFFICE")
+
+
+	def test_officespace_accepts_str(self):
+		self.assertRaises(ValueError, self.office_space.create_new, 111)
```
Comment (GDP254) [inline test_cases/tests.py]: Your function name is descriptive but, it does not match its contents in my opinion. I think `test_officespace_rejects_int` would be better.

**Comment by GDP254** [comment]
Git commits should be in imperative form. I found this section of the linked reference to be helpful:

> A properly formed Git commit subject line should always be able to complete the following sentence:
> 
>     If applied, this commit will your subject line here
> 
> For example:
> 
>     If applied, this commit will refactor subsystem X for readability
>     If applied, this commit will update getting started documentation
>     If applied, this commit will remove deprecated methods
>     If applied, this commit will release version 1.0.0
>     If applied, this commit will merge pull request #123 from user/branch
> 
> Notice how this doesn’t work for the other non-imperative forms:
> 
>     If applied, this commit will fixed bug with Y
>     If applied, this commit will changing behavior of X
>     If applied, this commit will more fixes for broken stuff
>     If applied, this commit will sweet new API methods
> 
>     Remember: Use of the imperative is important only in the subject line. You can relax this restriction when you’re writing the body.
> 

Source: https://chris.beams.io/posts/git-commit/

**Review by kevintumbo — APPROVED** [review]

**Comment by mnjaggah** [comment]
thanks for the link @GDP254  i totally missed the imperative nature of commit messages on that last one

**Review by maquchizi — APPROVED** [review]
This is good work. Please consider the following in addition to what your peers raised:
1. PEP8 coding standards for Python (line spacing especially)
2. The test file is getting quite large. Perhaps you could break it out into separate files for readability

**Pull request merged**